deploy/transport-controller-deployment.yaml.template

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: transport-controller-config
+  namespace: ${WDS_NAME}-system
+data:
+  get-kubeconfig.sh: |
+    #!/bin/bash
+    # this script receives a ControlPlane name and a parameter
+    # that determines whether to extract the ControlPlane's in-cluster kubeconfig
+    # or the external kubeconfig (if set to "true", the first will be retrieved).
+    # The function returns the requested kubeconfig's content in base64.
+    # it assumes that it runs within a kubeflex context.
```

Comment:
I am not sure what "a kubeflex context" means. Do you mean that `kubectl` is configured (in the usual ways, by `$KUBECONFIG` and `~/.kube/config`) to access the hosting cluster?

Reply:
yes, fixed.